clients/python/provider/superposition_provider/interfaces.py

```
@@ -0,0 +1,345 @@
+        """
+        try:
+            config = await self.resolve_all_features_async(evaluation_context)
+            if flag_key not in config:
+                return FlagResolutionDetails(default)
+
+            value = config[flag_key]
+            extracted = extractor(value)
+            if extracted is None:
+                return FlagResolutionDetails(default)
+            return FlagResolutionDetails(extracted)
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)